### ISSUE TITLE
404 not found in import section of dash

### ISSUE THREAD

**Issue by rcrath** [anvc/scalar#19]
hi, in my install of scalar (http://dahi.manoa.hawaii.edu), I get a 404 error in the import/export tab for the import function section of the page
![screenshot from 2015-09-09 11-30-08](https://cloud.githubusercontent.com/assets/944636/9774757/46f8214e-56e6-11e5-9269-af282752a6a0.png)

it is a fresh install from a day or so ago from git clone

**Comment by craigdietrich** [comment]
Hi Rich,

Thanks for your interest in Scalar over at Hawaiʿi DH!  

The Transfer Tool (that resides in an iframe there in the Import/Export tab) is actually kept in a seperate repository -- though it's linked in Scalar's Git, it doesn't download into the ZIP file.

All you need to do is head here: https://github.com/Michael628/scalar-book-transfer-tool , download the ZIP and place its contents into:

/system/application/plugins/transfer

Your timing is well, we just made a few bug fixes to the Transfer Tool.

Thanks,
Craig

**Comment by rcrath** [comment]
that did it.  that was fast! thanks.  I have a small DH grad class and we will be trying scalar this semester as an authoring platform.

**Comment by craigdietrich** [comment]
Great to hear, keep us posted :)

**Comment by ccc2lu** [comment]
I installed Scalar here at Lehigh University and ran into this issue as well.  I was about to open an issue when I found this closed one for the same problem.  It would probably be a good idea to add this to the installation documentation, since the "Import/Export" tab is there in core Scalar.  Either that or add some code to the Import/Export page to check if the transfer plugin is there, and if not put some text in saying it needs to be installed.  The "404 Page not found" error is quite confusing, heh.
